### PR TITLE
replace Java WxTimerModule with C++ setTimeout/setInterval implementa…

### PR DESCRIPTION
**Pull request by lycool**
replace Java WxTimerModule with C++ setTimeout/setInterval implementation.

clear all unhandled setTimeout/setInterval timers after Weex instance is destroyed

### CODE REVIEW

File: android/sdk/src/main/java/com/taobao/weex/bridge/WXBridgeManager.java [hunk 1]
```diff
@@ -570,7 +570,10 @@ public int callNative(String instanceId, String tasks, String callback) {
     if (WXEnvironment.isApkDebugable()) {
       mLodBuilder.append("[WXBridgeManager] callNative >>>> instanceId:").append(instanceId)
           .append(", tasks:").append(tasks).append(", callback:").append(callback);
-      WXLogUtils.d(mLodBuilder.substring(0));
+
+      //WXLogUtils.d(mLodBuilder.substring(0));
```
Comment (sospartan) [inline android/sdk/src/main/java/com/taobao/weex/bridge/WXBridgeManager.java]: Just remove it.

File: android/sdk/src/main/java/com/taobao/weex/bridge/WXBridgeManager.java [hunk 2]
```diff
@@ -1069,7 +1072,10 @@ public void invokeExecJS(String instanceId, String namespace, String function,
               .append("function:").append(function);
       if(logTaskDetail)
         mLodBuilder.append(" tasks:").append(WXJsonUtils.fromObjectToJSONString(args));
-      WXLogUtils.d(mLodBuilder.substring(0));
+
+      //WXLogUtils.d(mLodBuilder.substring(0));
```
Comment (sospartan) [inline android/sdk/src/main/java/com/taobao/weex/bridge/WXBridgeManager.java]: Just remove it.

Comment (lycool) [inline android/sdk/src/main/java/com/taobao/weex/bridge/WXBridgeManager.java]: WXLogUtils.d  removed

**Comment by lycool** [comment]
WXLogUtils.d

File: android/sdk/src/main/java/com/taobao/weex/bridge/WXBridgeManager.java (hunk 1 above)
Comment (lycool) [inline android/sdk/src/main/java/com/taobao/weex/bridge/WXBridgeManager.java]: WXLogUtils.d

Comment (lycool) [inline android/sdk/src/main/java/com/taobao/weex/bridge/WXBridgeManager.java]: WXLogUtils.d removed

**Pull request merged**